include/mbedtls/ccm.h

```diff
@@ -1,10 +1,20 @@
 /**
  * \file ccm.h
  *
- * \brief Counter with CBC-MAC (CCM) for 128-bit block ciphers
+ * \brief CCM combines Counter mode encryption with CBC-MAC authentication 
+ * for 128-bit block ciphers.
```

Comment:
Is the indentation deliberate here? I would have expected the sentence to continue at the level of `CCM combines ...`.

Reply:
fixed